instrumentation/graphql/README.md

```diff
@@ -0,0 +1,76 @@
+# OpenTelemetry GraphQL Instrumentation
+
+The OpenTelemetry GraphQL Ruby gem is a community maintained instrumentation for [GraphQL][graphql-home]. The GraphQL integration activates instrumentation for GraphQL queries.
+
+## How do I get started?
+
+Install the gem using:
+
+```
+gem install opentelemetry-instrumentation-graphql
+```
+
+Or, if you use [bundler][bundler-home], include `opentelemetry-instrumentation-graphql` in your `Gemfile`.
+
+## Usage
+
+To use the instrumentation, call `use` with the name of the instrumentation:
+
+```ruby
+OpenTelemetry::SDK.configure do |c|
+  c.use 'OpenTelemetry::Instrumentation::GraphQL'
+end
+```
+
+### Configuration options
+
+```ruby
+OpenTelemetry::SDK.configure do |c|
+  c.use 'OpenTelemetry::Instrumentation::GraphQL', {
+    # If this option is not provided the default behaviour is to instrument all
+    # schemas that extend GraphQL::Schema
+    schemas: [MyAppSchema],
+
+    # The following options are disabled by default as they generate
+    # large traces, but can be enabled using the following keys to add
+    # spans that provide more detail.
+    # Further details about what these keys correspond can be found on the [platform_tracing.rb class](https://github.com/rmosolgo/graphql-ruby/blob/1.10.x/lib/graphql/tracing/platform_tracing.rb#L28-L73).
+    # enable_platform_field maps to the execute_field and execute_field_lazy keys
+    enable_platform_field: true,
+    # enable_platform_authorized maps to the authorized and authorized_lazy keys
+    enable_platform_authorized: true,
+    # enable_platform_resolve_type maps to the resolve_type and resolve_type_lazy keys
+    enable_platform_resolve_type: true
```

Comment:
very small nit but maybe we should leave these commented out so we avoid ppl copy/pasting this snippet in and enabling stuff that, for the most part, we want disabled

Reply:
I see what you mean, I wanted to demonstrate how they could turn it on but I guess I can just as easily do that by provided a false value instead.  I'll give it an update.